Autopilot/AttitudeManager/timer.h

```diff
@@ -12,12 +12,12 @@
 /**
  * Number of Timer2 ticks in a millisecond. To calculate this, take:
  * 1/(frequencyCPU/Timer2PreScaler)*TICKS_TO_MSEC should equal close to 0.001 or 1 ms
- * In this case, (1/(41Mhz/64))*642 == ~1ms
+ * In this case, (1/(40 MHz/64))*625 == 1 ms
  * 
  * This value is used to accurately define the timer2 period (used for output pwm
  * frequency
  */
-#define T2_TICKS_TO_MSEC 642
+#define T2_TICKS_TO_MSEC 625
```

Comment:
You'd have to output MiddlePWM  or something and see what you expect (probably 1ms pulse) 

The value seems close to what it was, so I don't expect too much of a problem.